planner/core/plan_cost_ver2.go

```diff
@@ -448,13 +451,16 @@ func (p *PhysicalHashAgg) getPlanCostVer2(taskType property.TaskType, option *Pl
 	groupCost := groupCostVer2(option, inputRows, p.GroupByItems, cpuFactor)
 	hashBuildCost := hashBuildCostVer2(option, outputRows, outputRowSize, float64(len(p.GroupByItems)), cpuFactor, memFactor)
 	hashProbeCost := hashProbeCostVer2(option, inputRows, float64(len(p.GroupByItems)), cpuFactor)
+	startCost := newCostVer2(option, cpuFactor,
+		10*3*cpuFactor.Value,
```

Comment:
Please add some comments to explain the `10*3`.

Reply:
updated